### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by AlexandruDoda** [avocode/avocode#339]
Describe your problem here…

-----

```
Error: invalid stored block lengths
    at Zlib._handle.onerror (zlib.js:345:17)
```

**Comment by rychlis** [comment]
Hi, what's your OS and Avocode version? When does this error happen? Could you try installing Avocode 2.3.0 that was just released if it's still happening?

**Comment by AlexandruDoda** [comment]
Using 2.2.0 and El Capitan Beta, I'll install the the new version, thank you :)

**Comment by rychlis** [comment]
Does the new version work fine?

**Comment by AlexandruDoda** [comment]
Sorry, didn't get a chance to update yet. Just did now. Will let you know if any more crashes come up.
Thank you!

**Comment by rychlis** [comment]
Can i close the issue now or did you experience the problem again? Does the app work well afterwards or does it break some functionality?

**Comment by AlexandruDoda** [comment]
It works fine now, thanks!

**Comment by petrbrzek** [comment]
This error should be now fixed with the latest Avocode 2.5.3. Please try it and let us know.
